docs/README.md

```diff
@@ -282,6 +282,18 @@ if (!isCi) {
 }
 ```
 
+You can also use your own logging functions if needed:
+
+```js
+// prepare.js
+const { configure } = require('husky)
```

Comment:
I see your point about combined unit tests/documentation - there is a missing single quote in `require('husky)`